### PR TITLE
fix: properly kill child processes on Windows

### PR DESCRIPTION
**Pull request by mtraver**

### CODE REVIEW

**Review by jskeet — APPROVED** [review]
You might get someone else to review the actual implementation, but the aim looks good to me, thanks.

**Comment by mtraver** [comment]
> You might get someone else to review the actual implementation, but the aim looks good to me, thanks.

Ok thanks!

Jason, could you review the Go here?

**Review by squee1945 — APPROVED** [review]

File: client/local_test.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const testProgram = `package main
+
+import (
+	"fmt"
+	"time"
+)
+
+func main() {
+	fmt.Println("Hello from test program")
```
Comment (squee1945) [inline client/local_test.go:32]: Should this mention that it is sleeping for a period?

Comment (mtraver) [reply client/local_test.go]: Sure, done.

**Pull request merged**